src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            System.out.println(lottery.getRandomBall().toString());
```

Comment:
Can we write just `System.out.println(lottery.getRandomBall());`?

Reply:
I just cheked, it really did write same result. I tested what would be  printed without overrided toString() method and got printed just references of  objects. I got the idea that if classes have toString() method it will be called. Correct me if  i'm wrong.